hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/RSProcedureDispatcher.java

```diff
@@ -306,6 +307,10 @@ private boolean scheduleForRetry(IOException e) {
           serverName, e.toString(), numberOfAttemptsSoFar);
         return false;
       }
+      if (isSaslError(e)) {
```

Comment:
I think here we need to follow the same pattern with CallQueueTooBigException, only if this is the first try, we can make sure that the task has not been sent yet.

Reply:
sounds good @Apache9, addressed in the latest revision